lib/datasource/local/member_model_cache_item.dart

```diff
@@ -21,15 +21,19 @@ class MemberModelCacheItemAdapter extends TypeAdapter<MemberModelCacheItem> {
     reader.readByte();
```

Comment:
if you are changing the cache, you must also change the file name, so we don't get errors reading the old cache format

Reply:
amm just changing the file name and class?

Reply:
fixed by change members cache key thank you